<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wordInsertView</class>
 <widget class="QWidget" name="wordInsertView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="EnglishlineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>39</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="TranslatelineEdit_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>241</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>单词</string>
   </property>
   <property name="buddy">
    <cstring>EnglishlineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>翻译</string>
   </property>
   <property name="buddy">
    <cstring>TranslatelineEdit_2</cstring>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="insertWordButton">
      <property name="text">
       <string>插入单词</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>240</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="wordTypeRadio">
      <property name="text">
       <string>单词</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="preffixRadioType">
      <property name="text">
       <string>前缀</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="suffixRadioType">
      <property name="text">
       <string>后缀</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="bodyRadioType">
      <property name="text">
       <string>词根</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>单词类型</string>
   </property>
   <property name="buddy">
    <cstring>TranslatelineEdit_2</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>记忆方式</string>
   </property>
   <property name="buddy">
    <cstring>TranslatelineEdit_2</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="TranslatelineEdit_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>241</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>EnglishlineEdit</tabstop>
  <tabstop>TranslatelineEdit_2</tabstop>
  <tabstop>TranslatelineEdit_3</tabstop>
  <tabstop>wordTypeRadio</tabstop>
  <tabstop>preffixRadioType</tabstop>
  <tabstop>suffixRadioType</tabstop>
  <tabstop>bodyRadioType</tabstop>
  <tabstop>insertWordButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>TranslatelineEdit_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>EnglishlineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>wordInsertView</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
